### ISSUE TITLE
Microsoft Edge not showing invalid data popups

### ISSUE THREAD

**Issue by ramereth** [osuosl/streamwebs#482]
This is confusing users into thinking there is a problem with the site.

**Comment by baileysingleton** [comment]
Hey @ramereth, this is what happens when you try to submit an empty form on mac: 
![screen shot 2017-10-13 at 1 09 28 pm](https://user-images.githubusercontent.com/26613813/31564446-ab9b7bdc-b017-11e7-9407-8d92a97d9a54.png)

When you hit submit, the page refreshes and the red text is put in there. I think it works fine, it's not the same pop up style, but it at least is notifying the user.

**Comment by ramereth** [comment]
Looks like it worked, here's what I see now:

![virtualbox_windows 10_13_10_2017_14_34_56](https://user-images.githubusercontent.com/48997/31567330-da7b9188-b023-11e7-97a5-59a4d28c814f.png)

**Comment by hertelc** [comment]
@ramereth close as a duplicate of #483?

**Comment by ramereth** [comment]
@hertelc that's the PR that fixes this

**Comment by hertelc** [comment]
@baileysingleton  this task is tied to the validation changes, right?  I'm trying to get a sense of where we are.

**Comment by hertelc** [comment]
Review of this fix is blocked by #645

**Comment by baileysingleton** [comment]
I don't think this issue is relevant anymore. This was made for a long time ago when our validation was wonky. I believe it is much different now, and we don't need to worry about this.

**Comment by hertelc** [comment]
@baileysingleton I would still like this on my radar when I go and do the final Staging validation before production.

**Comment by hertelc** [comment]
This looks good in staging.  Closing.
